apis/api-journeys-modern/src/schema/event/chat/chatOpenEventCreate.mutation.ts

```diff
@@ -0,0 +1,73 @@
+import { prisma } from '@core/prisma/journeys/client'
+
+import { builder } from '../../builder'
+import { sendEventsEmail, validateBlockEvent } from '../utils'
+
+import { ChatOpenEventRef } from './chatOpenEvent'
+import { ChatOpenEventCreateInput } from './inputs'
+
+builder.mutationField('chatOpenEventCreate', (t) =>
+  t.withAuth({ isAuthenticated: true }).field({
+    override: {
+      from: 'api-journeys'
+    },
+    nullable: false,
+    type: ChatOpenEventRef,
+    args: {
+      input: t.arg({ type: ChatOpenEventCreateInput, required: true })
+    },
+    resolve: async (_parent, args, context) => {
+      const { input } = args
+      const userId = context.user?.id
+
+      if (!userId) {
+        throw new Error('User not authenticated')
+      }
+
+      const { visitor, journeyId, journeyVisitor } = await validateBlockEvent(
```

Comment:
I saw you removed this button, should it get removed here too?

```suggestion
      const { visitor, journeyId } = await validateBlockEvent(
```